### PR TITLE
Rearrange Timeline Layout

### PR DESCRIPTION
**Pull request by beardofedu**
## Overview
**TL;DR**
I didn't notice that the layout duplicated sides when reviewing this: #510 , so here is a fix for that.  

### Review
- @crichID since you created #510 
- @hectorsector since you helped review #510

### CODE REVIEW

File: _includes/timeline.html [hunk 1]
```diff
@@ -96,11 +96,11 @@
 
                         <li class="timeline-inverted">
                             <div>
-                                <a href="/on-demand/github-desktop/"><img class="timeline-image" src="./images/red-polo.png" alt=""></a>
+                                <a href="/on-demand/windows/"><img class="timeline-image" src="./images/red-polo.png" alt=""></a>
```
Comment (beardofedu) [inline _includes/timeline.html]: 🙇

**Comment by crichID** [comment]
@beardofedu did you update services-web with this change? Or would you like me to do it?

**Comment by beardofedu** [comment]
@crichID I haven't done it yet, but I would love to see you do it as a refresher.

**Comment by crichID** [comment]
> I haven't done it yet, but I would love to see you do it as a refresher.

@beardofedu let's hop on zoom tomorrow and make it happen :wink:

**Comment by beardofedu** [comment]
@crichID sounds good.

**Pull request merged**